### PR TITLE
CDS-88116-kubctl version update

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

https://harness.atlassian.net/browse/CDS-88116

### CODE REVIEW

**Review by abosii-harness — APPROVED** [review]

**Review by acasian-harness — APPROVED** [review]

File: docs/platform/delegates/delegate-reference/delegate-required-sdks.md [hunk 1]
```diff
@@ -121,6 +121,28 @@ mkdir -p /opt/harness-delegate/client-tools/tf-config-inspect/v1.1/ && curl -L h
...
+    value: |
+
+        ## for AKS
+        mkdir -m 777 -p client-tools/kubelogin/v0.1.1 \
+        && curl -s -L -o client-tools/kubelogin/v0.1.1/kubelogin https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/$TARGETARCH/kubelogin
+        export PATH=/opt/harness-delegate/client-tools/kubelogin/v0.1.1/:$PATH
+
+        ## for GKE or AKS with certificate auth type
+        mkdir -m 777 -p client-tools/harness-credentials-plugin/v0.1.0 \
+        && curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin
```
Comment (abosii-harness) [inline docs/platform/delegates/delegate-reference/delegate-required-sdks.md:143]: sorry, it's my bad when shared this script, replace this:
```
curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin
```
with:
```
curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin https://app.harness.io/public/shared/tools/harness-credentials-plugin/release/v0.1.0/bin/linux/amd64/harness-credentials-plugin
```

File: docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md [hunk 2]
```diff
@@ -225,6 +225,31 @@ To install multiple tools, you can add all the install scripts to the `INIT_SCRI
...
+    value: |
+
+        ## for AKS
+        mkdir -m 777 -p client-tools/kubelogin/v0.1.1 \
+        && curl -s -L -o client-tools/kubelogin/v0.1.1/kubelogin https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/$TARGETARCH/kubelogin
+        export PATH=/opt/harness-delegate/client-tools/kubelogin/v0.1.1/:$PATH
+
+        ## for GKE or AKS with certificate auth type
+        mkdir -m 777 -p client-tools/harness-credentials-plugin/v0.1.0 \
+        && curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin
```
Comment (abosii-harness) [inline docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:248]:
```
curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin
```
with:
```
curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin https://app.harness.io/public/shared/tools/harness-credentials-plugin/release/v0.1.0/bin/linux/amd64/harness-credentials-plugin
```

File: docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md [hunk 3]
```diff
@@ -225,6 +225,31 @@ To install multiple tools, you can add all the install scripts to the `INIT_SCRI
...
+If you're using a custom delegate with `kubelogin` and certificate type of authentication, then you must install Azure CLI. Alternatively, you can install the `harness-credentials-plugin` to take care of this flow without Azure CLI.
+:::
+
+```yaml
+  - name: INIT_SCRIPT
+    value: |
+
+        ## for AKS
+        mkdir -m 777 -p client-tools/kubelogin/v0.1.1 \
+        && curl -s -L -o client-tools/kubelogin/v0.1.1/kubelogin https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/$TARGETARCH/kubelogin
```
Comment (abosii-harness) [inline docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:243]: replace this:
```
https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/$TARGETARCH/kubelogin
```

with
```
https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/amd64/kubelogin
```

File: docs/platform/delegates/delegate-reference/delegate-required-sdks.md [hunk 4]
```diff
@@ -121,6 +121,28 @@ mkdir -p /opt/harness-delegate/client-tools/tf-config-inspect/v1.1/ && curl -L h
...
+
+You can replace the `harness-credentials-plugin` with Azure CLI or `gke-gcloud-auth-plugin`to take care of this flow. For more details, go to [Authentication in GKE v1.26](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke).
+
+```yaml
+  - name: INIT_SCRIPT
+    value: |
+
+        ## for AKS
+        mkdir -m 777 -p client-tools/kubelogin/v0.1.1 \
+        && curl -s -L -o client-tools/kubelogin/v0.1.1/kubelogin https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/$TARGETARCH/kubelogin
```
Comment (abosii-harness) [inline docs/platform/delegates/delegate-reference/delegate-required-sdks.md:138]: replace this:
```
https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/$TARGETARCH/kubelogin
```

with
```
https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/amd64/kubelogin
```

**Review by abosii-harness — CHANGES_REQUESTED** [review]

File: docs/platform/delegates/delegate-reference/delegate-required-sdks.md (hunk 4 above)
Comment (roshnisarangadharan) [inline docs/platform/delegates/delegate-reference/delegate-required-sdks.md:138]:
```suggestion
        && curl -s -L -o client-tools/kubelogin/v0.1.1/kubelogin https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/amd64/kubelogin
```

File: docs/platform/delegates/delegate-reference/delegate-required-sdks.md (hunk 1 above)
Comment (roshnisarangadharan) [inline docs/platform/delegates/delegate-reference/delegate-required-sdks.md:143]:
```suggestion
        && curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin https://app.harness.io/public/shared/tools/harness-credentials-plugin/release/v0.1.0/bin/linux/amd64/harness-credentials-plugin 
```

File: docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md (hunk 3 above)
Comment (roshnisarangadharan) [inline docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:243]:
```suggestion
        && curl -s -L -o client-tools/kubelogin/v0.1.1/kubelogin https://app.harness.io/public/shared/tools/kubelogin/release/v0.1.1/bin/linux/amd64/kubelogin
```

File: docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md (hunk 2 above)
Comment (roshnisarangadharan) [inline docs/platform/delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:248]:
```suggestion
        && curl -s -L -o client-tools/harness-credentials-plugin/v0.1.0/harness-credentials-plugin https://app.harness.io/public/shared/tools/harness-credentials-plugin/release/v0.1.0/bin/linux/amd64/harness-credentials-plugin 
```

**Review by abosii-harness — APPROVED** [review]

**Pull request merged**